core/src/main/scala/org/apache/spark/rdd/RDD.scala

```diff
@@ -1296,6 +1296,7 @@ abstract class RDD[T: ClassTag](
    * an exception if called on an RDD of `Nothing` or `Null`.
    */
   def take(num: Int): Array[T] = withScope {
+    val takeRampUpRate = conf.getInt("spark.rdd.take.rampUpRate", 4)
```

Comment:
Any pointers on what's the best way to do this? I haven't seen anything that's in core and sql. I can not introduce sqlconf option and just read sparkconf but then you lose the type safety. Thanks!